### ISSUE TITLE
plot(index, which="map", cex=0.5) uses small font for depth scale

### ISSUE THREAD

**Issue by dankelley** [ArgoCanada/argoFloats#542]
I won't bother with a reprex since the following images make things clear, and I'm sure this must come up with any index dataset.

**Case 1: default cex (looks good)**
```R
plot(indices[[1]], which="map")
```
<img width="793" alt="Screen Shot 2022-01-20 at 9 44 31 AM" src="https://user-images.githubusercontent.com/99469/150350921-7c8e9023-bf05-4674-b18d-63893d1a5601.png">

**Case 2: cex=0.5 (looks bad)**

```R
plot(indices[[1]], which="map", cex=0.5)
```
<img width="793" alt="Screen Shot 2022-01-20 at 9 44 48 AM" src="https://user-images.githubusercontent.com/99469/150350890-fed06b91-2ae4-4c62-bae0-b605799eb3a3.png">

**Comment by dankelley** [comment]
@j-harbin -- I'm going to go ahead and fix this.

You can see the problem in the snapshot.  We should not be handing `cex` to `drawPalette()`.  If a user wants to mess with the overall `cex` value, they should use `par(cex=...)`, as for other software.  

There are some subtleties to `cex`, as illustrated below, but I don't say that as an excuse.  I made an error, plain and simple.
```R
names(par())[grep("cex",names(par()))]
[1] "cex"      "cex.axis" "cex.lab"  "cex.main" "cex.sub" 
```

**Comment by dankelley** [comment]
Fixed in commit 44a00c6f58d83a2355b6d7f271bce24f0a17b175 of "develop" branch.
